src/main/java/gov/cms/madie/hqmf/HQMFFinalCleanUp.java

```diff
@@ -0,0 +1,198 @@
+package gov.cms.madie.hqmf;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import gov.cms.madie.hqmf.dto.MeasureExport;
+import javax.xml.xpath.XPathExpressionException;
+
+public class HQMFFinalCleanUp {
+
+  /** The Constant logger. */
+  private static final Logger logger = LoggerFactory.getLogger(HQMFFinalCleanUp.class);
+
+  /** The Constant reverseEntryCheckFile. */
+  private static final String reverseEntryCheckFile = "xsl/final_hqmf_entry_deletion_check.xsl";
+
+  /** The Constant deleteUnUsedEntryFile. */
+  private static final String deleteUnUsedEntryFile = "xsl/hqmf_delete_Unused_entry.xsl";
+
+  /**
+   * Clean.
+   *
+   * @param me the me
+   */
```

Comment:
Repetitive javadocs is how I know it came from the MAT codebase.

Reply:
me the me allover!